ui/components/multichain/notification-detail-nft/notification-detail-nft.stories.tsx

```diff
@@ -17,11 +18,13 @@ const Template = (args: NotificationDetailNftProps) => (
 
 export const DefaultStory = Template.bind({});
 DefaultStory.args = {
-  tokenSrc:
-    'https://i.seadn.io/s/raw/files/a96f90ec8ebf55a2300c66a0c46d6a16.png?w=500&auto=format',
-  networkSrc:
-    'https://token.api.cx.metamask.io/assets/nativeCurrencyLogos/ethereum.svg',
-  tokenId: 'NFT ID',
-  tokenName: 'NFT Name',
-  networkName: 'Ethereum',
+  nft: {
+    address: '0xAddress',
+    chainId: '0x1',
+    image:
+      'https://i.seadn.io/s/raw/files/a96f90ec8ebf55a2300c66a0c46d6a16.png?w=500&auto=format',
```

Comment:
@MetaMask/notifications when did our notification NFT information start failing to render!

This was before this PR (on develop), investigate what changed.

![Screenshot 2024-10-25 at 09 43 35](https://github.com/user-attachments/assets/f2e4097f-8356-4af4-83b4-116bc6184516)

Reply:
I'm resolving this for our notifications team to investigate.